terraform/cp4aiops/outputs.tf

```diff
@@ -41,5 +41,5 @@ output "cp4aiops_password" {
 
 // Namespace
 output "namespace" {
-  value = var.namespace
+  value = var.cp4aiops_namespace
```

Comment:
I don't think this is necessary to list in outputs since this is a user-defined variable listed in `variables.tf`.

Reply:
It has been removed.